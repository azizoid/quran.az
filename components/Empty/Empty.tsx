import Link from 'next/link'
import { Card } from '../../ui/Card/Card'

export const Empty = (): JSX.Element => (
  <div className="space-y-4">
    <ol className="py-2 w-full flex justify-center text-gray-500 space-x-5 text-sm md:text-base">
      <li>
        <Link href="/1">
          <a>Fatihə surəsi</a>
        </Link>
      </li>
      <li>
        <Link href="/36">
          <a>Ya-sin surəsi</a>
        </Link>
      </li>
      <li>
        <Link href="/55">
          <a>Ər-Rəhman surəsi</a>
        </Link>
      </li>
      <li>
        <Link href="/67">
          <a>Əl-Mülk surəsi</a>
        </Link>
      </li>
      <li>
        <Link href="/2/255">
          <a>Ayətul-Kürsi</a>
        </Link>
      </li>
    </ol>

    <div className="col">
      <h6 className="alert alert-success">
        Axtarışınızın <strong>uğurlu</strong> olması üçün aşağıdakı{' '}
        <strong>qaydalara</strong> riayət edin:
      </h6>
    </div>

    <div className="grid grid-1 md:grid-cols-3 gap-2 py-4">
      <Card title="	İstifadə qaydaları" size="small">
        quran.az/<code>[surə]</code>
        <br />
        quran.az/<code>[surə]</code>/<code>[ayə]</code>
        <br />
        quran.az/search/<code>[kəlmə]</code>
      </Card>
      <Card title="Hərf səhvləri" size="small">
        Axtarış zamanı etdiyiniz qrammatik səhvlər sözlərin tapılmamasına səbəb
        ola bilər
        <br />
        Məsələn: <code>mekke</code> əvəzinə <code>Məkkə</code> sözünü axtarın
      </Card>
      <Card title="Fərqli söz və tərcümə" size="small">
        Axtardığınız fikir tərcümədə olmaya bilər; <br />
        Müəlliflərdən qaynaqlanaraq tərcümələrdəki sözlər və fikirlər fərqli ola
        bilər
      </Card>
    </div>
  </div>
)

export default Empty
