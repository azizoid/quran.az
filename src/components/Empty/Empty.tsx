import Link from 'next/link'

import { Card } from '@/ui'
import { SoorahList } from 'src/components'

export const Empty = (): JSX.Element => (
  <div className="space-y-4">
    <ol className="py-2 w-full flex justify-center text-gray-500 space-x-5 text-sm md:text-base">
      <li>
        <Link href="/1">Fatihə surəsi</Link>
      </li>
      <li>
        <Link href="/36">Ya-sin surəsi</Link>
      </li>
      <li>
        <Link href="/55">Ər-Rəhman surəsi</Link>
      </li>
      <li>
        <Link href="/67">Əl-Mülk surəsi</Link>
      </li>
      <li>
        <Link href="/2/255">Ayətul-Kürsi</Link>
      </li>
    </ol>

    <SoorahList />

    <div className="col">
      <h6 className="alert alert-success">
        Axtarışınızın <strong>uğurlu</strong> olması üçün aşağıdakı <strong>qaydalara</strong>{' '}
        riayət edin:
      </h6>
    </div>

    <div className="grid grid-1 md:grid-cols-3 gap-2 py-4">
      <Card title="	İstifadə qaydaları" size="small">
        quran.az/<code>[surə]</code>
        <br />
        quran.az/<code>[surə]</code>/<code>[ayə]</code>
        <br />
        quran.az/search/<code>[kəlmə]</code>
      </Card>
      <Card title="Hərf səhvləri" size="small">
        Axtarış zamanı etdiyiniz qrammatik səhvlər sözlərin tapılmamasına səbəb ola bilər
        <br />
        Məsələn: <code>mekke</code> əvəzinə <code>Məkkə</code> sözünü axtarın
      </Card>
      <Card title="Fərqli söz və tərcümə" size="small">
        Axtardığınız fikir tərcümədə olmaya bilər; <br />
        Müəlliflərdən qaynaqlanaraq tərcümələrdəki sözlər və fikirlər fərqli ola bilər
      </Card>
    </div>
  </div>
)
