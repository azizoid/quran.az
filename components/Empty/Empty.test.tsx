import React from 'react'
import {render }from '@testing-library/react'
import {Empty} from './Empty'

test("Empty Snapshot", () =>{
  const {container} = render(<Empty />)

  expect(container).toMatchSnapshot()
})