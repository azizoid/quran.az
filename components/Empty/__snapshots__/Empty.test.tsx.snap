// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty Snapshot 1`] = `
<div>
  <div
    class="space-y-4"
  >
    <ol
      class="py-2 w-full flex justify-center text-gray-500 space-x-5 text-sm md:text-base"
    >
      <li>
        <a
          href="/1"
        >
          Fatihə surəsi
        </a>
      </li>
      <li>
        <a
          href="/36"
        >
          Ya-sin surəsi
        </a>
      </li>
      <li>
        <a
          href="/55"
        >
          Ər-Rəhman surəsi
        </a>
      </li>
      <li>
        <a
          href="/67"
        >
          Əl-Mülk surəsi
        </a>
      </li>
      <li>
        <a
          href="/2/255"
        >
          Ayətul-Kürsi
        </a>
      </li>
    </ol>
    <div
      class="col"
    >
      <h6
        class="alert alert-success"
      >
        Axtarışınızın 
        <strong>
          uğurlu
        </strong>
         olması üçün aşağıdakı
         
        <strong>
          qaydalara
        </strong>
         riayət edin:
      </h6>
    </div>
    <div
      class="grid grid-1 md:grid-cols-3 gap-2 py-4"
    >
      <div
        class="card"
      >
        <div
          class="card-title px-4 undefined"
        >
           İstifadə qaydaları
        </div>
        <div
          class="card-content px-4 undefined"
        >
          quran.az/
          <code>
            [surə]
          </code>
          <br />
          quran.az/
          <code>
            [surə]
          </code>
          /
          <code>
            [ayə]
          </code>
          <br />
          quran.az/search/
          <code>
            [kəlmə]
          </code>
        </div>
      </div>
      <div
        class="card"
      >
        <div
          class="card-title px-4 undefined"
        >
          Hərf səhvləri
        </div>
        <div
          class="card-content px-4 undefined"
        >
          Axtarış zamanı etdiyiniz qrammatik səhvlər sözlərin tapılmamasına səbəb ola bilər
          <br />
          Məsələn: 
          <code>
            mekke
          </code>
           əvəzinə 
          <code>
            Məkkə
          </code>
           sözünü axtarın
        </div>
      </div>
      <div
        class="card"
      >
        <div
          class="card-title px-4 undefined"
        >
          Fərqli söz və tərcümə
        </div>
        <div
          class="card-content px-4 undefined"
        >
          Axtardığınız fikir tərcümədə olmaya bilər; 
          <br />
          Müəlliflərdən qaynaqlanaraq tərcümələrdəki sözlər və fikirlər fərqli ola bilər
        </div>
      </div>
    </div>
  </div>
</div>
`;
